### ISSUE TITLE
Reimplement (or drop) misc_flak and forgotten props

### ISSUE THREAD

**Issue by ETLegacyTeam** [etlegacy/etlegacy#949]
Author: @IR4T4 (IR4T4)
Date: 2016-09-23
Redmine Issue: 949, https://dev.etlegacy.com/issues/949

---

There are still code fragments from rtcw of misc_flak.

**Comment by ETLegacyTeam** [comment]
From: @realkemon (keMoN)
Date: 2016-09-23T11:29:56Z

---

Didn't you want to implement an operable Flak /AA gun in its place? I think it would be fantastic to have ischbinz' "2cm Flak gun":http://forums.warchest.com/showthread.php/31520-2cm-Flak-model-preview operable and deal damage to vehicles. Basically like the AT gun in Venice, however completely functional like MGs.

**Comment by realkemon** [comment]
I'm just going to hijack this ticket to expand it to other only partly supported entities. A full list can be found here: https://forums.splashdamage.com/t/forgotten-props/120388

Aciz provided a test map for `props_snowGenerator` in Discord.
![8crqYfS](https://user-images.githubusercontent.com/8881420/88278529-d1a2bd00-cce2-11ea-95fb-29025a9957e1.png)
https://streamable.com/3s0wbj
[props_snowGenerator.zip](https://github.com/etlegacy/etlegacy/files/4965421/props_snowGenerator.zip)

**Comment by rmarquis** [comment]
Yep. Extra note: we should eventually take the official ET mapping guide at https://wiki.splashdamage.com/index.php/Wolfenstein:_Enemy_Territory and update it along our [own doc](https://github.com/etlegacy/etlegacy-docs), and documenting the above.
